### PR TITLE
 home page 

### PR DESCRIPTION
**Pull request by Samju200**
home page 1
home page 2

You can just choose the career path you want to work it and style it

### CODE REVIEW

File: cyber.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```
Comment (netwox) [inline cyber.html:1]: Can you try to format your code such that it's easier to track the tags

File: style.css [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    margin: 0;
+    padding: 0;
+
+}
+body {
+    margin: 0 auto;
+    padding: 0;
+    font-size: 18px;
+    line-height: 18px;
+    font-family: oswald PT sans serif;
```
Comment (netwox) [inline style.css:14]: I think this should be
`font-family: Oswald, PT Sans, serif;`

File: web.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+</head>
+<body>
+    <header>
+        <div class="header-link">
+        <div class="logo"><h1>NILE</h1></div>
+        <div class="nav-link">
+        <nav>
+            <ul>
+                <a href="index.html"><li> Home</li></a>
+                <a href=""><li>Contanct Us</li></a>
```
Comment (netwox) [inline web.html:17]: Dead link.

File: web.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="header-link">
+        <div class="logo"><h1>NILE</h1></div>
+        <div class="nav-link">
+        <nav>
+            <ul>
+                <a href="index.html"><li> Home</li></a>
```
Comment (netwox) [inline web.html:16]: the Structure for this should `<li> <a href="index.html">Home</a></li>`
with `li` tag encompassing the `a` tag

File: data.html [hunk 5]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Data Scientist</title>
```
Comment (netwox) [inline data.html:6]: I think this should be **Data Science**

File: index.html [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+<body>
+    <header>
+        <div class="header-link">
+        <div class="logo"><h1>NILE</h1></div>
+        <div class="nav-link">
+        <nav>
+            <ul>
+                <a href="index.html"><li> Home</li></a>
+                <a href=""><li>Contanct Us</li></a>
+                <a href=""><li> <button>Form</button> </li></a>
```
Comment (netwox) [inline index.html:18]: There's no need the `button` tag here.

**Review by opeyemipeter1759 — APPROVED** [review]

**Comment by Israel-Force** [comment]
It's responsive ryt? @Samju200

**Comment by Israel-Force** [comment]
And I think using white colour for the header text and Logo will them come out a little better... I might do that once this merged sha.

**Review by Israel-Force — APPROVED** [review]

**Comment by Samju200** [comment]
> It's responsive ryt? @Samju200

@Israel-Force not responsitive yet

**Comment by Samju200** [comment]
> It's responsive ryt? @Samju200

yes responsive now

**Pull request merged**